src/remote-utils.ts

```diff
@@ -0,0 +1,50 @@
+import fetch from 'node-fetch';
+import { rootCACertPath } from '../src/constants';
+import { Agent } from 'https';
+import * as fs from 'fs';
+
+/**
+ * Returns the remote box's certificate
+ * @param hostname - hostname of the remote machine
+ * @param port - port to connect the remote machine
+ *
+ * @public
+ */
+export async function getRemoteCertificate(
+  hostname: string,
+  port: number
+): Promise<string> {
+  const agent = new Agent({
+    ca: fs.readFileSync(rootCACertPath, { encoding: 'utf-8' })
```

Comment:
I would break this out a bit since there are several things that could go wrong and it's inconvenient when debugging to have them all on the same line.
- Build the path to the CA cert
- Read it 
- Pass it into the agent

I think that at least the first two should probably be top-level `const` declarations in this module